graphyte.py

```diff
@@ -51,8 +54,12 @@ def __init__(self, host, port=2003, prefix=None, timeout=5, interval=None,
         self.protocol = protocol
         self.batch_size = batch_size
         self.tags = tags
+        self.raise_send_errors = raise_send_errors
 
         if self.interval is not None:
+            if raise_send_errors:
+                logger.info('Raising of errors will be disabled in asynchronous mode')
+                self.raise_send_errors = False
```

Comment:
Yeah, I think I'd prefer a hard failure here (`ValueError`). It seems to me calling it that way is basically wrong, and shouldn't go unnoticed.

Reply:
I have no issue with that.  Do you have preferred verbiage? 

I stubbed in "raise_send_errors must be disabled when interval is set" but can change before I commit.

Reply:
Yep, that works for me!